### ISSUE TITLE
How to specify the prefixURL with AMD loaders

### ISSUE THREAD

**Issue by avandecreme** [openseadragon/openseadragon#792]
See the comments there: http://stackoverflow.com/questions/34152450/example-of-using-require-to-use-openseadragon/34158414?noredirect=1#comment56076434_34158414

I am not sure whether we have to do something about it.

Maybe @homerjam has an idea?

**Comment by homerjam** [comment]
Sorry I'm not sure. But wouldn't you get a reference to the main class and then just create a viewer as indicated in the SO post?

```
var viewer = OpenSeadragon({
  prefixUrl: 'url',
  ...
});

**Comment by iangilman** [comment]
I think the hope is that the prefixUrl wouldn't have to "poke deep into the modules directory space"...This is of particular interest if the server isn't serving the node_modules folder (which generally you wouldn't). 

I suppose one option would be to include all the images in the JS code as data URIs.

**Comment by homerjam** [comment]
Yes, ideally these would be SVGs too. And the ability to skin via CSS would be helpful.

Webpack enables you to require images/css etc and have them converted into data URIs in the bundle it spits out. I'm not sure if thats something that can be implemented without breaking other things though?

**Comment by iangilman** [comment]
SVG buttons and CSS skinning should get their own issues, I suppose. 

I don't know anything about webpack, but it sounds like maybe it's worth looking into, if anyone wants to pursue this.

**Comment by leesei** [comment]
I would copy `./node_modules/openseadragon/build/openseadragon/images/` to the `dist` folder in my build step.

**Comment by avandecreme** [comment]
So I converted the [OpenSeadragonFiltering plugin](https://github.com/NIST-ISG/OpenSeadragonFiltering) demo to webpack.
As @leesei recommended, I added a copy step using the `CopyWebPack` plugin:
https://github.com/NIST-ISG/OpenSeadragonFiltering/blob/master/webpack.config.js#L32-L38

**Comment by adrianq** [comment]
Hi Guys! Any update on this? Shall we still copy the images during the build phase or is there a better approach?

**Comment by iangilman** [comment]
Nothing beyond what you see in this thread. Any suggestions?

**Comment by adrianq** [comment]
Sorry for the late response. I finally followed [this example](https://openseadragon.github.io/examples/ui-binding-custom-buttons/) and created a toolbar with font-awesome.

**Comment by iangilman** [comment]
@adrianq Nice! Seems like a good solution.
